zh_gsdsimp-ud-train.conllu

```diff
@@ -57101,10 +57103,10 @@
 23	。	。	PUNCT	.	_	5	punct	_	SpaceAfter=No
 
 # sent_id = train-s2069
-# text = 在JR鲹沢駅下车后再乘车30分钟。
+# text = 在JR鲹泽驿下车后再乘车30分钟。
 1	在	在	VERB	VV	_	4	acl	_	SpaceAfter=No
 2	JR	JR	X	FW	_	1	obj	_	SpaceAfter=No
-3	鲹沢駅	鲹沢駅	PROPN	NNP	_	2	flat:foreign	_	SpaceAfter=No
+3	鲹泽驿	鲹泽驿	PROPN	NNP	_	2	flat:foreign	_	SpaceAfter=No
```

Comment:
This is actually a complex case IMO, because although 駅 correspond to the simplified character 驿, it is no longer used in this context in modern Chinese. So I think it makes sense to either keep the whole thing in Japanese (i.e., the original token), or we should probably write it as `鲹泽 站`, i.e. two separate tokens.
Wikipedia left it alone in this case, but I don't think that's necessarily the gold standard in this particular case (https://zh.wikipedia.org/wiki/%E7%99%BD%E7%A5%9E%E5%B1%B1%E5%9C%B0), because it is also not uncommon for news reports to translate 駅.

Reply:
Yes, in my original idea I left it 鯵ヶ沢駅 as its original Japanese name, but ヶ seemed very bad...

Reply:
I've tentatively changed it into JR鲹泽站 (also changed Wikipedia to make link to https://zh.wikipedia.org/zh-hans/鰺澤站).

Reply:
I think this is great! Thank you!